templates/hoover.nomad

```diff
@@ -87,6 +87,40 @@ job "hoover" {
     }
   }
 
+  group "tika" {
+    task "tika" {
+      driver = "docker"
+      config {
+        image = "logicalspark/docker-tikaserver"
```

Comment:
Please tag this image with the latest fixed tag from https://hub.docker.com/r/logicalspark/docker-tikaserver/tags